app/src/main/java/com/yacgroup/yacguide/TourbookActivity.kt

```diff
@@ -113,30 +113,41 @@ class TourbookActivity : BaseNavigationActivity() {
     }
 
     private fun _import(uri: Uri) {
-        val confirmDialog = Dialog(this)
-        confirmDialog.setContentView(R.layout.dialog)
-        confirmDialog.findViewById<TextView>(R.id.dialogText).text = getString(
-                R.string.override_tourbook)
-        confirmDialog.findViewById<View>(R.id.yesButton).setOnClickListener {
-            try {
-                TourbookExporter(_db, contentResolver).importTourbook(uri)
-                Toast.makeText(this, getString(R.string.tourbook_import_successfull),
-                        Toast.LENGTH_SHORT).show()
-            } catch (e: Exception) {
-                Toast.makeText(this, getString(R.string.tourbook_import_error),
-                        Toast.LENGTH_SHORT).show()
+        val confirmDialog = AlertDialog.Builder(this).apply {
+            setTitle(R.string.warning)
+            setMessage(getString(R.string.override_tourbook))
+            setIcon(android.R.drawable.ic_dialog_alert)
+            setNegativeButton(R.string.cancel) { dialog, _ -> dialog.dismiss()}
+            setPositiveButton(R.string.ok) { dialog, _ ->
+                try {
+                    TourbookExporter(_db, contentResolver).importTourbook(uri)
+                    _makeToast(getString(R.string.tourbook_import_successfull))
+                } catch (e: Exception) {
+                    _showImportError(e.message.toString())
+                }
+                _initYears()
             }
-            _initYears()
-            confirmDialog.dismiss()
         }
-        confirmDialog.findViewById<View>(R.id.noButton).setOnClickListener {
-            confirmDialog.dismiss()
-        }
-        confirmDialog.setCanceledOnTouchOutside(false)
-        confirmDialog.setCancelable(false)
         confirmDialog.show()
     }
 
+    private fun _showImportError(errMsg: String) {
+        val dialog = AlertDialog.Builder(this).apply {
```

Comment:
I think, this dialog makes sense. In a follow-up PR (issue #210) I would suggest to add a button here to directly file a bug report with the given error. I would furthermore suggest to also add #210 to milestone v0.3.0 and make the release afterwards.

- [ ] Add this ToDo to #210 description.
- [ ] Set milestone of #210 to v0.3.0